sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -476,15 +476,14 @@ for idx, doc in enumerate(docs):
             for data_source in entity.data_sources:
                 print("......Entity ID: {}".format(data_source.entity_id))
                 print("......Name: {}".format(data_source.name))
-        if len(entity.related_entities) > 0:
-            print("...Related Entities:")
-            for related_entity, relation_type in entity.related_entities.items():
-                print("......Entity Text: {}".format(related_entity.text))
-                print("......Relation Type: {}".format(relation_type))
+    for relation in doc.relations:
+        print("Relation of type: {} has the following roles".format(relation.relation_type))
+        for role in relation.roles:
+            print("...Role '{}' with entity '{}'".format(role.name, role.entity.text))
```

Comment:
I think so, yes. I purposely didn't no `normalize_text` in this pr, wanted to keep it to `relations` as much as possible. But good point, in the pr i do for `normalized_text`, I'll make the samples print that instead of text